### PR TITLE
Test UI with playwright

### PR DESCRIPTION
**Pull request by gc153671**
An initial test using playwright for testing UI elements. I'll contribute docs on how to run the tests in another PR

### CODE REVIEW

**Comment by hamelin** [comment]
So are we comfortable with closing this PR then? Or is there value in merging it, and only the docs will come in a separate PR?

File: thisnotthat/tests/conftest.py [hunk 1]
```diff
@@ -0,0 +1,47 @@
+import pytest
```
Comment (hamelin) [inline thisnotthat/tests/conftest.py:1]: @gclendenning do you know the purpose of this, or did you rip this out of the Panel unit tests? Asking from curiosity, I would love to know whether this is strictly necessary.

File: thisnotthat/tests/ui/test_datapane.py [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+        'str': ['A', 'B', 'C', 'D'],
+        'bool': [True, True, True, False],
+        'date': [dt.date(2019, 1, 1), dt.date(2020, 1, 1), dt.date(2020, 1, 10), dt.date(2019, 1, 10)],
+        'datetime': [dt.datetime(2019, 1, 1, 10), dt.datetime(2020, 1, 1, 12), dt.datetime(2020, 1, 10, 13), dt.datetime(2020, 1, 15, 13)],
+    })
+    df.index.name = 'row_num'
+    return df
+
+
+def test_datapane_selection(page, df_mixed):
```
Comment (hamelin) [inline thisnotthat/tests/ui/test_datapane.py:32]: I looked up the unit test report from the CI pipeline, and could not get confirmation this test was effectively run. Were you able to ascertain this?

**Review by hamelin — COMMENTED** [review]
Looks good, but would like some clarification before we go ahead and merge.

File: thisnotthat/tests/conftest.py (hunk 1 above)
Comment (gc153671) [reply thisnotthat/tests/conftest.py:1]: This was taken out of the panel unit tests to make the UI tests optional. They will only run with the --ui option

```
pytest thisnotthat/tests/ --ui --browser chromium -n 1
```

File: thisnotthat/tests/ui/test_datapane.py (hunk 2 above)
Comment (gc153671) [reply thisnotthat/tests/ui/test_datapane.py:32]: It ran locally and I was able to successfully run other Panel tests in a separate devops pipeline.

**Comment by gc153671** [comment]
I've added some documentation to the README but let me know if there is a better place for it.

**Review by hamelin — APPROVED** [review]
This is great!

**Pull request merged**